generated/alerting/api/openapi.yaml

```diff
@@ -3627,5 +3627,5 @@ components:
       type: http
     apiKeyAuth:
       in: header
-      name: ApiKey
+      name: Authorization
```

Comment:
Running, for example, `make generate-alerting-client` resets this to `ApiKey` which is incorrect. The head is `Authorization` and the prefix is `ApiKey`. Probably needs to be fixed upstream?

Reply:
https://github.com/elastic/kibana/pull/171496

Reply:
No, I don't mean to block anything here, was just making sure you didn't also fix it upstream.